### PR TITLE
CORE-1101: Merge 'master-brd-app' into 'develop'

### PR DESCRIPTION
**Pull request by EBGToo**

### CODE REVIEW

**Review by EBGToo — COMMENTED** [review]
Note to self: Examining major differences.

File: WalletKitCore/src/crypto/handlers/btc/BRCryptoExportablePaperWalletBTC.c [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+static BRCryptoExportablePaperWalletStatus
+cryptoExportablePaperWalletValidateSupporteBTC (BRCryptoNetwork network,
+                                                BRCryptoCurrency currency) {
+    return CRYPTO_EXPORTABLE_PAPER_WALLET_SUCCESS;
+}
+
+static BRCryptoAddress
+cryptoExportablePaperWalletCreateAddressBTC (BRCryptoNetwork network,
+                                             BRCryptoKey key) {
+    return NULL;
```
Comment (ewerx) [inline WalletKitCore/src/crypto/handlers/btc/BRCryptoExportablePaperWalletBTC.c:26]: unintentional?

Comment (EBGToo) [reply WalletKitCore/src/crypto/handlers/btc/BRCryptoExportablePaperWalletBTC.c]: I added a view.

**Review by carlcherry — APPROVED** [review]
Not sure how to review all these changes - but I know that they are already in production.

**Review by ewerx — APPROVED** [review]

**Pull request merged**